### PR TITLE
Run rspec for forked repos

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

File: tests/spec/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Rspec tests
```
Comment (juliogonzalez) [inline tests/spec/README.md]: This should be at doc/TESTS.md

**Comment by juliogonzalez** [comment]
Other than the doc stuff, LGTM :-D

File: tests/spec/README.md (hunk 1 above)
Comment (MalloZup) [reply tests/spec/README.md]: yop ok thx! :)

File: doc/TESTS.md [hunk 2]
```diff
@@ -20,13 +20,20 @@ rake test
 
 These tests use [rspec](http://rspec.info/) are not executed on travis, so you need to launch the execution manually.
 
-You can use the following command:
+
+## Run your tests always on your forked repo!
```
Comment (juliogonzalez) [inline doc/TESTS.md]: This shouldn't be a title but something like:

```WARNING```: Run your tests always on your forked repo!

Comment (MalloZup) [reply doc/TESTS.md]: ok

File: doc/TESTS.md [hunk 3]
```diff
@@ -20,13 +20,20 @@ rake test
...
+
+
+You need a fixed PR open for running some tests.
+
+Repo is the value of your forked repo
```
Comment (juliogonzalez) [inline doc/TESTS.md]: I'd give first the syntax, then an example:
==============================================

Syntax:
```console
 +repo=<REPO> pr_num=<PR_NUM> rspec -fd cmdline_spec.rb
```

Where ```REPO``` is your repository in format ```project/repo````and ```PR_NUM``` is an opened and mergeable Pull Request.

For example:

```console
 cd tests/spec/
repo=MalloZup/gitarro pr_num=1 rspec -fd cmdline_spec.rb
```

**Pull request merged**